docs/archive/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
It's odd that this file would be created. Pretty sure someone else created it earlier today. Is it possible that there are two similarly-named files/folders?

Reply:
Ah yes @gshafiei used `Archive` instead. I would rename your folder so there's only one.

Reply:
Done!